pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -26,6 +26,21 @@ import (
 	"github.com/openshift/origin/pkg/image/importer"
 )
 
+const (
+	// DockerRegistryURLEnvVar is a mandatory environment variable name specifying external url of internal
+	// docker registry. All references to pushed images will be prefixed with its value.
+	DockerRegistryURLEnvVar = "DOCKER_REGISTRY_URL"
+
+	// EnforceQuotaEnvVar is an boolean environment variable that allows to turn quota enforcement on or off.
+	// By default, quota enforcement is off. Recognized values are "true", and "false".
+	EnforceQuotaEnvVar = "OPENSHIFT_ENFORCE_QUOTA"
+
+	// ProjectCacheTTLEnvVar is an environment variable specifying an eviction timeout for project quota
```

Comment:
Unfortunately, middleware options cannot be overriden by environment variables. We can support configuration via config file for sure, but we need to allow for easy overrides as well unless we start generating a config file as a secret during `oadm registry`.

I'll allow for configuration with the config file but still allow for overrides with these variables.